torch_np/_ndarray.py

```diff
@@ -167,7 +188,10 @@ def __iadd__(self, other):
 
     def __sub__(self, other):
         other_tensor = asarray(other).get()
-        return asarray(self._tensor.__sub__(other_tensor))
+        try:
+            return asarray(self._tensor.__sub__(other_tensor))
+        except RuntimeError as e:
+            raise TypeError(e.args)
```

Comment:
Why here and not everywhere else?

Reply:
This is gone in the next PRs in this stack.